styles/google/catppuccin.user.css

```diff
@@ -0,0 +1,1093 @@
+/* ==UserStyle==
+@name           Google Catppuccin Less
```

Comment:
Name should just be `Google Catppuccin` without the `Less` part.

Reply:
Ah yeah this was a mistake, I added the "Less" part just so I don't get confused between it and the old Stylus one during development

Reply:
Gotcha, nw.